### ISSUE TITLE
[1.0] Parameter information not shown for chained functions

### ISSUE THREAD

**Issue by dlsniper** [go-lang-plugin-org/go-lang-idea-plugin#1342]
When there are a few functions changed, it seems that the parameter information is not shown for ```.Methods```

I'll try and come up with a better example tomorrow
```go
package main

type (
    ds1 struct {
    }

    ds2 struct {

    }
)

func (d ds1) d1(param1 string, param2 int) ds2 {
    return ds2{}
}

func (d ds2) d2(param1 string, param2 int) int {
    return 1
}

func main() {
    d := ds1{}
    d.d1("1", 2).d2("1", 2)
}
```

**Comment by ignatov** [comment]
Please add more detailed sample.

**Comment by dlsniper** [comment]
I've updated the example.

Also, I've noticed something strange in the autocomplete when trying to write the line: ```d.d1("1", 2).d2//autocomplete```, see the screenshot below. I'n not sure if it's the related to another ticket I've seen or not (I'll try and see if I can find the related).

![snapshot3](https://cloud.githubusercontent.com/assets/607868/6313499/8e7b1df8-b9ae-11e4-8974-cac3fd6fce9c.png)
